src/components/dateTimePicker/index.js

```diff
@@ -98,7 +105,11 @@ class DateTimePicker extends Component {
     /**
      * Render custom input
      */
-    renderInput: PropTypes.elementType
+    renderInput: PropTypes.elementType,
+    /**
+     * Override system theme variant (dark or light mode) used by the date picker.
+     */
+    themeVariant: PropTypes.oneOf(Object.values(THEME_VARIANTS))
```

Comment:
This is a required prop? Can it be an optional one?

Reply:
> This is a required prop? Can it be an optional one?

It is optional. Required would be `themeVariant: PropTypes.oneOf(Object.values(THEME_VARIANTS)).isRequired`